vowpalwabbit/cb_adf.cc

```diff
@@ -525,12 +549,16 @@ base_learner* cb_adf_setup(options_i& options, vw& all)
   all.label_type = label_type::cb;
 
   cb_adf* bare = ld.get();
-  learner<cb_adf, multi_ex>& l = init_learner(ld, base, CB_ADF::do_actual_learning<true>,
-      CB_ADF::do_actual_learning<false>, problem_multiplier, prediction_type::action_scores);
+  learner<cb_adf, multi_ex>& l = init_learner(ld, base, learn,
+      predict, problem_multiplier, prediction_type::action_scores);
   l.set_finish_example(CB_ADF::finish_multiline_example);
 
   bare->gen_cs.scorer = all.scorer;
 
-  l.set_save_load(CB_ADF::save_load);
+  if (all.model_file_ver >= VERSION_FILE_WITH_CB_ADF_SAVE)
```

Comment:
This change broke the TestSharedModel test under cs_unitttest @jackgerrits , I'll take a look later. This might mean that it gets updated in some other way.

Reply:
Something related to learner.h line 255

Reply:
Weird, okay let's take a look today